app/src/main/java/com/douglasbruce/tasky/features/task/navigation/TaskNavigation.kt

```diff
@@ -20,6 +21,9 @@ internal const val taskDateArg = "taskDate"
 @VisibleForTesting
 internal const val taskIsEditingArg = "taskIsEditing"
 
+private const val DEEP_LINK_TASK_URI_PATTERN =
+    "https://www.tasky.com/task/{$taskDateArg}/{$taskIsEditingArg}?$taskIdArg={$taskIdArg}"
```

Comment:
This URL doesn't exist. For deep links that aren't backed by a real URL, you can use a custom scheme, so for example `tasky://task/{$taskDateArg}...`